### DISCUSSION TITLE
无法初始化程序

### DISCUSSION THREAD

**Discussion by MCSeekeri** [spencerwooo/onedrive-vercel-index#310] [Q&A] [answered]
[复现站点在此](https://mcsg-vercel.vercel.app/)

第二步就卡住了,已经获取到了http://localhost/?*** 的链接,但粘贴后程序仍无反应。

已经尝试过关闭所有广告拦截插件,隐私模式下操作,切换浏览器等多种方案仍然无效。

使用自己的应用替换了默认应用。Repo地址[在此](https://github.com/MCSeekeri/onedrive-vercel-index)

**Comment by spencerwooo** [comment, marked as answer]
> ……，但粘贴后程序仍无反应。

具体指什么？

**Reply by MCSeekeri** [reply]
获取完地址填进去...
![image](https://user-images.githubusercontent.com/20928094/150959319-439fd3d6-a03f-40ad-b399-abe8e71fa48f.png)
然后我就开始等待（
然后什么都没发生

**Reply by zkeq** [reply]
> 获取完地址填进去... ![image](https://user-images.githubusercontent.com/20928094/150959319-439fd3d6-a03f-40ad-b399-abe8e71fa48f.png) 然后我就开始等待（ 然后什么都没发生

试着将`https`换成`http` ?

**Reply by MCSeekeri** [reply]
> > 获取完地址填进去... ![image](https://user-images.githubusercontent.com/20928094/150959319-439fd3d6-a03f-40ad-b399-abe8e71fa48f.png) 然后我就开始等待（ 然后什么都没发生
> 
> 试着将`https`换成`http` ?

......竟然有用了...

我没想到HTTPS Everywhere竟然会把localhost也给加一个https
我自裁。

**Reply by spencerwooo** [reply]
懂了，算 bug 吧，后面改一下提取机制
